OpenLaTeXStudio/EditorModule/src/latexstudio/editor/EditorTopComponent.java

```diff
@@ -82,21 +83,26 @@ public void propertyChange(PropertyChangeEvent evt) {
                     autoCompletion.setAutoActivationDelay((Integer) evt.getNewValue());
                 } else if (evt.getPropertyName().equals(ApplicationSettings.LINEWRAP_STATUS) && rSyntaxTextArea != null) {
```

Comment:
I wanted to bring the mechanism to the master branch ASAP in order for everyone to use it. Migration of the other settings is up next.